.github/workflows/test-notify-slack.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Run notify-slack action'
+
+on:
+  pull_request:
+    paths:
+      - actions/notify-slack/*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test-notify-slack-action:
+    name: 'Uses the notify-slack action'
+    runs-on: ubuntu-latest
+    if: always()
+    outputs:
+      status: ${{ steps.notify.outputs.status }}
+    steps:
+      - name: 'Test notify-slack action'
+        id: notify
+        uses: shopsmart/github-actions/actions/notify-slack@feature/slack-notify
+        with:
+          application: github-actions
+          environment: staging
+          status: ${{ needs.jobs.outputs.status }}
+          type: deployment
+          version: v1.0.56
+          template: ./github-actions/actions/notify-slack/message.json.j2
```

Comment:
```suggestion
          template: ./.github/actions/test-notify-slack/message.json.j2
```

Reply:
Not sure what the `./github-actions` is meant to be since checkout will checkout to the current directory.  For testing purposes, we should point to a different message.json.j2.